packages/common/src/HospitalUtils.ts

```diff
@@ -9,8 +9,18 @@ export const activeHospitals = [
   Hospital.TEL_HASHOMER,
 ];
 
-export function getAllHospitalOptions(defaultLabel?: string) {
-  return getHospitalOptions(activeHospitals, defaultLabel);
+export const ALL_HOSPITALS_SELECT = "ALL";
+
+export function getAllHospitalOptions(
```

Comment:
1. I don't think we need to take the label as an argument, but we need a flag to know whether to put the "all" and the "" options or not.
2. Not all users should see the option "All" , only those who have permissions to more than 1 hospital.

All in all, you can resolve this by just checking if `hospitals.length > 1` and if so, add the desired option.

I think you can _hard code_ the labels, no need to pass them.

Reply:
it will break lots of code in different places that use this method. I think it is better to do this in a separate PR.

Reply:
So what I did is for:
   1. I changed it to be a hard coded value, but only for the getALl function..
   2. I moved the check of more then one hospital into the getAll function (it was in the screen file)